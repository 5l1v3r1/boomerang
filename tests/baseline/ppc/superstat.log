Unknown library function __xstat
after removing matching assigns (%pc).
in proc stat adding addrExp r1{0} - 32 to local table
in proc stat adding addrExp r1{0} + 4 to local table
finding final parameters for stat
after removing matching assigns (%pc).
in proc main adding addrExp r1{0} - 128 to local table
in proc main adding addrExp r1{0} - 124 to local table
in proc main adding addrExp r1{0} - 116 to local table
in proc main adding addrExp r1{0} - 112 to local table
in proc main adding addrExp r1{0} - 108 to local table
in proc main adding addrExp r1{0} - 104 to local table
in proc main adding addrExp r1{0} - 100 to local table
in proc main adding addrExp r1{0} - 96 to local table
in proc main adding addrExp r1{0} - 92 to local table
in proc main adding addrExp r1{0} - 84 to local table
in proc main adding addrExp r1{0} - 80 to local table
in proc main adding addrExp r1{0} - 76 to local table
in proc main adding addrExp r1{0} - 72 to local table
in proc main adding addrExp r1{0} - 64 to local table
in proc main adding addrExp r1{0} - 56 to local table
in proc main adding addrExp r1{0} - 144 to local table
in proc main adding addrExp r1{0} - 4 to local table
in proc main adding addrExp r1{0} + 4 to local table
in proc main adding addrExp r1{0} - 136 to local table
in proc main adding addrExp r1{0} - 132 to local table
in proc main adding addrExp r1{0} - 32 to local table
### WARNING: iteration limit exceeded for dfaTypeAnalysis of procedure main ###
in proc main adding addrExp r1{0} - 128 to local table
in proc main adding addrExp r1{0} - 124 to local table
in proc main adding addrExp r1{0} - 116 to local table
in proc main adding addrExp r1{0} - 112 to local table
in proc main adding addrExp r1{0} - 108 to local table
in proc main adding addrExp r1{0} - 104 to local table
in proc main adding addrExp r1{0} - 100 to local table
in proc main adding addrExp r1{0} - 96 to local table
in proc main adding addrExp r1{0} - 92 to local table
in proc main adding addrExp r1{0} - 84 to local table
in proc main adding addrExp r1{0} - 80 to local table
in proc main adding addrExp r1{0} - 76 to local table
in proc main adding addrExp r1{0} - 72 to local table
in proc main adding addrExp r1{0} - 64 to local table
in proc main adding addrExp r1{0} - 56 to local table
in proc main adding addrExp r1{0} - 144 to local table
in proc main adding addrExp r1{0} - 4 to local table
in proc main adding addrExp r1{0} + 4 to local table
in proc main adding addrExp r1{0} - 136 to local table
in proc main adding addrExp r1{0} - 132 to local table
in proc main adding addrExp r1{0} - 32 to local table
in proc main adding addrExp r1{0} - 116 to local table
in proc main adding addrExp r1{0} - 112 to local table
in proc main adding addrExp r1{0} - 108 to local table
in proc main adding addrExp r1{0} - 104 to local table
in proc main adding addrExp r1{0} - 100 to local table
in proc main adding addrExp r1{0} - 84 to local table
in proc main adding addrExp r1{0} - 80 to local table
in proc main adding addrExp r1{0} - 76 to local table
in proc main adding addrExp r1{0} - 72 to local table
in proc main adding addrExp r1{0} - 64 to local table
in proc main adding addrExp r1{0} - 56 to local table
finding final parameters for stat
local g3 has type char * that doesn't agree with type of define void of a library, why?
local g4 has type /*signed?*/int that doesn't agree with type of define void of a library, why?
local g4 has type void that doesn't agree with type of define /*signed?*/int of a library, why?
local g4 has type /*signed?*/int that doesn't agree with type of define void of a library, why?
local g4 has type void that doesn't agree with type of define __size32 of a library, why?
