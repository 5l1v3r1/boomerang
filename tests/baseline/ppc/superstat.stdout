Boomerang alpha 0.3.2 19/Aug/2010
loading...
decoding entry point...
decoding anything undecoded...
finishing decode...
found 2 procs
decompiling...
 considering main
  considering stat
  decompiling stat
 decompiling main
global type analysis for main
global type analysis for stat
generating code...
output written to ./tests/outputs/ppc/superstat
completed in 0 secs.
